CLI/src/main/java/main/Drifty_CLI.java

```diff
@@ -55,12 +58,41 @@ public static void main(String[] args) {
             link = null;
             String name = null;
             String location = null;
+            yamlFilePath = Program.get(Program.DRIFTY_PATH) + File.separator + YAML_FILENAME;
             for (int i = 0; i < args.length; i++) {
                 switch (args[i]) {
                     case HELP_FLAG, HELP_FLAG_SHORT -> {
                         help();
                         System.exit(0);
                     }
+                    case ADD_FLAG -> {
+                        if (i + 1 >= args.length) {
+                            messageBroker.msgInitError("No URL provided.");
+                            System.exit(1);
+                        }
+                        addUrlToFile(args[i + 1]);
+                        System.exit(0);
+                    }
+                    case LIST_FLAG -> {
+                        listUrls();
+                        System.exit(0);
+                    }
+                    case REMOVE_FLAG -> {
+                        if (i + 1 >= args.length) {
+                            messageBroker.msgInitError("No line number provided for removal.");
```

Comment:
@Femi-lawal Please consider using `Batch` as the `MessageCategory` that is you would use **`msgBatchInfo()`** and **`msgBatchError()`** methods in the whole process of printing output logs for `add`, `remove` and `list` arguments.